spread-shellcheck

```diff
@@ -191,6 +242,7 @@ def is_file_in_dirs(file, dirs):
 
     return False
 
+
```

Comment:
I noticed you added an extra newline at the bottom of some methods. Is this a Python thing to make the separation between methods clearer?

Reply:
Yes, it's a convention, see https://pep8.org/#blank-lines